prow/githuboauth/githuboauth.go

```diff
@@ -160,14 +165,19 @@ func (ga *GithubOAuthAgent) HandleRedirect(client OAuthClient, getter GithubClie
 			ga.serverError(w, "Save session", err)
 			return
 		}
-
 		ghc := getter.GetGithubClient(token.AccessToken, false)
 		user, err := ghc.GetUser("")
 		if err != nil {
 			ga.serverError(w, "Get user login", err)
 			return
 		}
-		http.SetCookie(w, &http.Cookie{Name: "github_login", Value: *user.Login})
+		http.SetCookie(w, &http.Cookie{
+			Name:    "github_login",
+			Value:   *user.Login,
+			Path:    "/",
+			Expires: time.Now().Add(time.Hour * 24 * 30),
```

Comment:
this probably ought to be configurable in the future. we don't have any super major reasons to expire logout, though we may want to add an explicit logout.